scripts/skills/effects/rf_cheap_trick_effect.nut

```diff
@@ -0,0 +1,99 @@
+this.rf_cheap_trick_effect <- ::inherit("scripts/skills/skill", {
+	m = {
+		// Config
+		HitChanceModifier = 0,		// Must be set during initialization
+		DamageRegularMult = 1.0,	// Must be set during initialization
+	},
+	function create()
+	{
+		this.m.ID = "effects.rf_cheap_trick";
+		this.m.Name = "Cheap Trick";
+		this.m.Description = "Unleash a quick and deceptive move to catch opponents off-guard, allowing for a higher chance of landing a hit but at the cost of power.";
+		this.m.Icon = "ui/perks/rf_cheap_trick.png";
```

Comment:
Do you not add an IconMini for this?

Reply:
I do not and I don't see a reason to do it for this effect.
This effect only lasts during the turn of the active entity, it is always removed once you give control over to the next entity.

Reply:
Sure, that's fine.